lib/ui/window/pointer_data_packet.cc

```diff
@@ -17,7 +18,19 @@ PointerDataPacket::PointerDataPacket(uint8_t* data, size_t num_bytes)
 PointerDataPacket::~PointerDataPacket() = default;
 
 void PointerDataPacket::SetPointerData(size_t i, const PointerData& data) {
+  FML_DCHECK(i < GetLength());
   memcpy(&data_[i * sizeof(PointerData)], &data, sizeof(PointerData));
 }
 
+PointerData PointerDataPacket::GetPointerData(size_t i) const {
+  FML_DCHECK(i < GetLength());
```

Comment:
DCHECK only runs in debug mode. if the goal of this PR is to add bounds checking to make it safe, you need to always perform the bounds check

Reply:
I added DCHECK because originally there is no check at all and I dare not change semantics of the code (e.g. if no check because previous dev deliberately do so to avoid being too slow?). If you like, I will change to CHECK